waku/v2/node/waku_node.nim

```diff
@@ -71,6 +70,7 @@ const clientId* = "Nimbus Waku v2 node"
 # Default Waku Filter Timeout
 const WakuFilterTimeout: Duration = 1.days
 
+const wakuNode2VersionString* = "version / git commit hash: " & git_version
```

Comment:
It's not followed everywhere, but we want constants to be in `PascalCase` (the exception are for those overridden by compiler definitions, such as `git_version` above).
This would just be `WakuNodeVersionString` (not WakuNode2)